### PR TITLE
NullableTypeTransformer - fix for attributes

### PR DESCRIPTION
**Pull request by kubawerlos**
Fixes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5533

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

**Comment by OskarStark** [comment]
It looks like this does not fix my case.

It works well without a default value, but that's the case before this fix too 🤔 

Can you extend your test case like proposed in my comment?

File: tests/Fixer/Operator/TernaryToElvisOperatorFixerTest.php [hunk 1]
```diff
@@ -498,4 +498,25 @@ public function provideFix70Cases()
...
+        $this->doTest($input);
+    }
+
+    public function provideDoNotFix80Cases()
+    {
+        return [
+            ['<?php
+
+function test(#[TestAttribute] ?User $user) {}
+'],
```
Comment (OskarStark) [inline tests/Fixer/Operator/TernaryToElvisOperatorFixerTest.php:519]:
```suggestion
            ['<?php

function test(#[TestAttribute] ?User $user) {}
'],
            ['<?php

function test(#[TestAttribute] ?User $user = null) {}
'],
```

**Comment by kubawerlos** [comment]
@OskarStark added, since in https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5533 you haven't added a config I can only guess, let's go with this one and verify if you still have the issue.

Might be another bug lurking at us.

**Comment by OskarStark** [comment]
Yes

**Review by keradus — APPROVED** [review]

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**